include/openssl/cmp.h

```diff
@@ -217,7 +224,12 @@ typedef struct OSSL_cmp_certresponse_st OSSL_CMP_CERTRESPONSE;
 DEFINE_STACK_OF(OSSL_CMP_CERTRESPONSE)
 typedef STACK_OF(ASN1_UTF8STRING) OSSL_CMP_PKIFREETEXT;
 
+#  define OPENSSL_NO_OCSP /* TODO remove when chunk 8 is contributed */
```

Comment:
Why is this needed?

Reply:
I've added this to provisionally disable the use of `OSSL_CMP_MSG_http_perform` in `cmp_ctx.c` until that function becomes available in chunk 8.